src/devices/CharacterLcd/samples/CharacterLcd.Sample.cs

```diff
@@ -17,11 +20,12 @@ class Program
         static void Main(string[] args)
         {
             // Sets up a 16x2 character LCD with a hardwired or no backlight.
-            using (Lcd1602 lcd = new Lcd1602(registerSelectPin: 22, enablePin: 17, dataPins: new int[] { 25, 24, 23, 18 }))
-            {
-                lcd.Clear();
-                lcd.Write("Hello World");
-            }
+            //using (Lcd1602 lcd = new Lcd1602(registerSelectPin: 22, enablePin: 17, dataPins: new int[] { 25, 24, 23, 18 }))
```

Comment:
why is this commented out?

Reply:
Only one example can be used at a time (it's highly unlikely one has all possible variants of an LCD display attached at the same time) and I changed this to work with mine... 
I'll put this into a functions, so that people testing (or playing with) their displays just need to switch a single line.

Reply:
Cleaned up, I hope this is more clear now.